02.extensions/zooport/02.troubleshooting/02.faq/docs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Does the standard ZOO Import/Export support ZOOlanders elements?"** - No, this is currently unsupported. 
+
+* **What should be the date format in a CSV file?** - The correct date format is YYYY-MM-DD HH:MM:SS. If a CSV file won't import, please make sure you are using the correct date format.
+
+* **Is it possible to import multiple Select element values?** - Yes, it is possible with CSV import. All possible options should be specified in the Select element settings. In the CSV file should be a separate column for each option, which is equal to the options' Values, not Names. 
```

Comment:
Changed the answer and added images. But Vika should help me with uploading images.